picard/util/tags.py

```diff
@@ -140,22 +219,96 @@ def insert(self, index, tagvar):
     def __repr__(self):
         return f"TagVars({self._items!r})"
 
-    def display_name(self, name):
-        tagdesc = None
+    def item_from_name(self, name):
         if ':' in name:
             name, tagdesc = name.split(':', 1)
+        else:
+            tagdesc = None
+
+        if name and name.startswith('_'):
+            search_name = name.replace('_', '~', 1)
+        elif name and name.startswith('~'):
+            search_name = name
+            name = name.replace('~', '_')
+        else:
+            search_name = name
+
+        item: TagVar = self._name2item.get(search_name, None)
+
+        return name, tagdesc, search_name, item
+
+    def display_name(self, name):
+        name, tagdesc, search_name, item = self.item_from_name(name)
 
-        item = self._name2item.get(name, None)
         if item and item.shortdesc:
             title = _(item.shortdesc)
         else:
-            title = name
-
+            title = search_name
         if tagdesc:
             return '%s [%s]' % (title, tagdesc)
         else:
             return title
 
+    @staticmethod
+    def _add_section(title, text):
+        return f"<p><strong>{title}</strong> {'; '.join(text)}.</p>"
+
+    def display_tooltip(self, name):
+        name, tagdesc, _search_name, item = self.item_from_name(name)
+
+        title = _(item.longdesc) if item and item.longdesc else _(TEXT_NO_DESCRIPTION)
+
+        if markdown is None:
+            title = '<p>' + title.replace('\n', '<br />') + '</p>'
+        else:
+            title = markdown(title)
+
+        notes = tuple(item.notes()) if item else tuple()
+        if notes:
+            title += self._add_section(_(TEXT_NOTES), notes)
+
+        if tagdesc:
+            return f"<p><em>%{name}%</em> [{tagdesc}]</p>{title}"
+        else:
+            return f"<p><em>%{name}%</em></p>{title}"
+
+    def display_full_description(self, name):
+        name, tagdesc, _search_name, item = self.item_from_name(name)
+
+        title = _(item.longdesc) if item and item.longdesc else _(TEXT_NO_DESCRIPTION)
```

Comment:
`title` isn't the best name for that imho, as it contains a whole block of text, what about `text`, `content` or something like that instead?

Reply:
Changed to `content` which better describes the variable.